packages/buttons/src/views/Anchor.js

```diff
@@ -31,33 +32,69 @@ const StyledAnchor = styled.a.attrs({
       [ButtonStyles['is-selected']]: selected
     })
 })`
+  ${props =>
+    props.external &&
+    `
+    &&& {
```

Comment:
Is this really necessary?

Reply:
I was able to remove a single level of specificity, now `&&`. Not sure how/why our CSS classes are so specific.